src/cli/useInteractionReducer.ts

```diff
@@ -73,7 +73,10 @@ function backendCommitGraphToDisplayCommits(
     hasFork: boolean,
   ): DisplayCommit[] {
     const childCommits = subgraphBackendRootCommit.childCommits.map(
-      (childCommitHash) => repository.commits[childCommitHash],
+      /*  SEE - Changes from Object Ref to Map.get, making the retutn type Commit | undefined. 
+      Adding a `!` here to get rid of TS errors
+      */
```

Comment:
This makes sense, but I think it'll be better if the comment explains the code rather than the change (because once this PR is merged the comment won't make sense)
```suggestion
      // Since the commits came from the graph, childCommitHash must exist in repository.commits.
```